ci/meta.yml

```diff
@@ -0,0 +1,79 @@
+meta:
+  plan:
+    terraform-bootstrap:
+      task: terraform-bootstrap
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: dwpdigital/jinja-yaml-aws
+            version: 0.0.19
+            tag: 0.0.19
+        run:
+          path: sh
+          args:
+            - -exc
+            - |
+              python bootstrap_terraform.py
+              cp terraform.tf ../terraform-config
```

Comment:
I think all of the terraform repos also render a tfvars file that will need `sed` to remove the assume role and then copy to output

Reply:
It's just empty, there as a template.